AsyncDisplayKit/ASImageNode.mm

```diff
@@ -181,7 +183,8 @@ - (NSObject *)drawParametersForAsyncLayer:(_ASDisplayLayer *)layer
                                                     opaque:self.opaque
                                              contentsScale:self.contentsScaleForDisplay
                                            backgroundColor:self.backgroundColor
-                                               contentMode:self.contentMode];
+                                               contentMode:self.contentMode
```

Comment:
Hmm, it looks like the work to port away from using the draw parameters for these core, extremely performance sensitive objects was never quite completed. I believe where we left off was the ability to implement instance methods, And we are using some instance variables in the method, but it seems at least some of the remaining work did not go through.

In addition, we should check the locking especially in the text implementation.